packages/react-core/src/components/Tabs/examples/TabsHelp.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Tabs, Tab, TabTitleText, TabAction, Popover } from '@patternfly/react-core';
+import HelpIcon from '@patternfly/react-icons/dist/esm/icons/help-icon';
+
+export const TabsHelp: React.FunctionComponent = () => {
+  const [activeTabKey, setActiveTabKey] = React.useState<number>(0);
+  const tabComponentRef = React.useRef<any>();
+  const firstMount = React.useRef(true);
```

Comment:
Can these be removed? It looks like they might have only been used for the Dynamic example to aid in focus management when adding/removing a tab.